### PR TITLE
Added the option to change the color of the message pane (issue -#238 )

### LINKED ISSUE #238: Adding the option to change the color of the message pane
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
No feature to change color of the message pane

**Expected behavior:**
To allow custom colors for the message pane

**Would you like to work on the issue?**

No

### PR DESCRIPTION
**Pull request by arnav-mandal1234**
Fixes #238

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added an option to change the color of the message pane.

#### Changes proposed in this pull request:

-change in the CSS and HTML files of the setting page
-change in the script.js and allowmic.js

susi

### CODE REVIEW

**Comment by arnav-mandal1234** [comment]
@stealthanthrax  @zamhaq @gabru-md @ms10398  please check into this.

File: src/js/allowmic.js [hunk 1]
```diff
@@ -29,14 +32,33 @@ preDefThemes.addEventListener("click",(e) => {
...
+			console.log("success");
+			alert("success");
+      });
+  }
 
+if(submit2) {
+			submit2.addEventListener("click",()=>{
+			localStorage.setItem("msgTheme",msgTheme);
+			console.log("success");
+			alert("success");
```
Comment (stealthanthrax) [inline src/js/allowmic.js]: remove the console logs, rest LGTM 🚀

Comment (arnav-mandal1234) [reply src/js/allowmic.js]: done.

**Comment by ms10398** [comment]
Thanks!
Please try to squash commits from next time accepted as its your first contributor

**Pull request merged**